packages/ffe-tables-react/src/Table.md

```diff
@@ -126,3 +126,202 @@ const onSort = ({sortBy, descending, tableData}) => {};
     onSort={onSort}
 />;
 ```
+Bruk av render funksjoner for columns: cellRender, columnHeaderRender, columnFooterRender.
+
+For hele tabellen: rowRender, headerRender, footerRender.
+
+```js
+const classNames = require('classnames');
+const HakeIkon = require('@sb1/ffe-icons-react/lib').HakeIkon;
+const KryssIkon = require('@sb1/ffe-icons-react/lib').KryssIkon;
+
+
+const generateCheckbox = (value) => {
+    return <div style={{width: '100%', textAlign: 'center'}}>{value ?
+        <HakeIkon className={classNames({'icon-green': true, 'ffe-table__expand-icon': true})}/> :
+        <KryssIkon className={classNames({'icon-red': true, 'ffe-table__expand-icon': true})}/>}</div>;
+};
```

Comment:
Husk å kjøre prettier på koden din!

Reply:
Also - her trenger du vel ikke `classNames`?

```js
<HakeIkon className="icon-green ffe-table__expand-icon" /> // etc
```

Reply:
I always run prettier! :) But it seems to avoid the .md files, have to just reformat outside of the file I guess.

Yes the class names are maybe too much of an example, I will remove them.

Reply:
@selbekk prettiefied and cleaned up